starter-core/src/main/java/io/micronaut/starter/feature/camunda/Platform7.java

```diff
@@ -23,17 +23,21 @@
 import io.micronaut.starter.feature.Category;
 import io.micronaut.starter.feature.FeatureContext;
 import io.micronaut.starter.feature.database.DatabaseDriverFeature;
+import io.micronaut.starter.feature.server.Jetty;
+import io.micronaut.starter.feature.server.Netty;
 import io.micronaut.starter.feature.test.AssertJ;
 import jakarta.inject.Singleton;
 
 @Singleton
 public class Platform7 implements CamundaCommunityFeature {
 
-    private final AssertJ assertJ;
     private final DatabaseDriverFeature defaultDbFeature;
+    private final Jetty jetty;
+    private final AssertJ assertJ;
 
-    public Platform7(DatabaseDriverFeature defaultDbFeature, AssertJ assertJ) {
+    public Platform7(DatabaseDriverFeature defaultDbFeature, Jetty jetty, AssertJ assertJ) {
```

Comment:
Annotate the new constructor with @Inject. Keep the old constructor deprecated.

Reply:
I don't see why we need to keep the old constructor. IMHO it would never be called. We need the new constructor to initialize all fields.